packages/hydrated_cubit/lib/src/hydrated_storage.dart

```diff
@@ -126,4 +140,16 @@ class HydratedStorage implements Storage {
     }
     return null;
   }
+
+  /// Induces manual box compaction. It's rarely needed.
+  /// Consider using custom compaction strategy instead.
+  ///
+  /// `compact` is [HydratedStorage] specific feature,
+  /// to use it downcast `Storage` to `HydratedStorage`.
+  Future<void> compact() {
+    return _box.isOpen ? _lock.synchronized(_box.compact) : Future.value();
+  }
 }
```

Comment:
Thoughts on avoiding exposing the `compact` API?

Reply:
> Thoughts on avoiding exposing the `compact` API?

I think every Bitcask should have compaction api